envoy/tcp/async_tcp_client.h

```diff
@@ -42,6 +42,8 @@ class AsyncTcpClient {
    * Connect to a remote host. Errors or connection events are reported via the
    * event callback registered via setAsyncTcpClientCallbacks(). We need to set the
```

Comment:
Since we removed the `callbacks_ = nullptr`, we can change this comments?

Reply:
Done, tnx